src/js/game/LevelMVC/LevelView.js

```diff
@@ -1104,7 +1106,9 @@ module.exports = class LevelView {
           (this.player.inventory[blockType] || 0) + 1;
         sprite.kill();
         this.toDestroy.push(sprite);
-        completionHandler();
+        if (completionHandler !== null) {
```

Comment:
all three of these `null` checks should just be `if (completionHandler)`; otherwise other values (for example, `undefined`) won't get handled correctly

Reply:
Roger that 👍